File: CC++/Assignments/Assignment 6/Divisible.c
/* Not divisible by 2 and 3 */

#include <stdio.h>
int main()
{

    int i;
    for (i = 1; i <= 100; i++)
        if (i % 2 != 0 && i % 3 != 0)
            printf("%d\t", i);
    return 0;
}

File: Java/Assignment 1/Divisible.java
import java.util.Scanner;

public class Divisible {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.print("Enter value for integer M : ");
        int M = in.nextInt();
        System.out.print("Enter value for integer N (>M) : ");
        int N = in.nextInt();
        if (N < M) {
            System.out.println("Invalid Input");
            System.exit(0);
        }
        int sum = 0;
        for (int i = M; i <= N; i++) {
            if (i % 15 == 0) {
                System.out.println(i);
                sum += i;
            }

        }
        System.out
                .println("Sum of all integers b/w " + M + "and " + N + " that are divisible by both 3 and 5 : " + sum);

        in.close();
    }
}